readme.md

```diff
@@ -60,3 +65,13 @@ In code:
 ### Demo
 
 See <https://norkart.github.io/L.Control.NorkartSearch/demo/> for a simple example
+
+
+### Contribute
+
+##### Install dependencies:
+```npm install```
+
+##### Run code:
+Run the demo code by running:
+```webpack-dev-server``` and go to ```localhost:8080```
```

Comment:
Could you add this to the "start"-script in package.json? Pretty standard to run npm start to start projects, nobody reads readmes. 

In addition, i cannot find webpack-dev-server in devDependencies?

(and, I could not get it to work by installing webpack-dev-server either

Reply:
True, added it to a start script now. 
But could you not just run the command webpack-dev-server? Without installing anything? That's what I did, and pretty sure I don't have a global install. Did you make sure you didn't have anything else running on port 8080?

Reply:
Nope, does not work for me. This is not a blocker though, we can fix this later